test/test_math_ops.cpp

```diff
@@ -855,6 +855,27 @@ TEST(MathOps, Cast2D) {
   opexecuter.RunTest();
 }  // end of test op Cast
 
+// Test op: Cos
+TEST(MathOps, Cos) {
+  Scope root = Scope::NewRootScope();
+  int dim1 = 2;
+  int dim2 = 4;
+
+  Tensor A(DT_FLOAT, TensorShape({dim1, dim2}));
+
+  AssignInputValues(A, 3.8f);
```

Comment:
This is testing for all values = 3.8f. For mathematical functions we should probe it at different points that might fail.
I'd suggest use `void AssignInputValues(Tensor& A, T x)` https://github.com/tensorflow/ngraph-bridge/blob/sarkars/resize_bilinear/test/test_utilities.h#L84

and test for `{0, -0, pi/2, pi, 1.0, 3.8, 4.2, -3.9, ...(some other float values)... , -1.0, std::numeric_limits<float>::infinity(), -std::numeric_limits<float>::infinity(), std::numeric_limits<float>::quiet_NaN(), std::numeric_limits<float>::signaling_NaN()}`

Reply:
infinity does not work, it fails while comparing TF and ngraph values. i updated the inputs with the other values.